lib/cfndsl/jsonable.rb

```diff
@@ -23,7 +23,7 @@ def FnFindInMap(map, key, value)
 
     # Equivalent to the CloudFormation template built in function Fn::GetAtt
     def FnGetAtt(logical_resource, attribute)
-      Fn.new('GetAtt', [logical_resource, attribute])
+      Fn.new('GetAtt', [logical_resource, attribute], [logical_resource])
```

Comment:
The third arg to Fn.new apparently existed for this purpose but was previously unused. I use it now where functions refer to other resources or parameters.